### PR TITLE
Home: Add a Homepage Component

### PR DESCRIPTION
**Pull request by HWaruguru**
Add a home component to make creation of tests easy

Task: [T327236](https://phabricator.wikimedia.org/T327236)

### CODE REVIEW

File: src/views/HomeView.vue [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <script setup>
 import { ref } from "vue";
+import Home from "../components/Home.vue";
 
-const tasks = ref({
```
Comment (Damilare1) [inline src/views/HomeView.vue:4]: Depends on what the backend would return in this situation. I'd assume its a list of tasks that a user can scroll between using the "Skip to Next" button. I'm thinking maybe we want to leave it as tasks and also make it an array of multiple tasks than a single task. Then we also need to model how that skip to next button would work.

Comment (blancadesal) [reply src/views/HomeView.vue:4]: > I'd assume its a list of tasks that a user can scroll between using the "Skip to Next" button. I'm thinking maybe we want to leave it as tasks and also make it an array of multiple tasks than a single task.

This is how I've been imagining it. A GET request to `api/tasks/` would return a batch of tasks, say 10, where a "batch" is a json object containing an array.

Comment (HWaruguru) [reply src/views/HomeView.vue:4]: Oh I see instead of 10 different requests we bundle them into one request which would lessen the runtime

**Comment by Damilare1** [comment]
Hi @HWaruguru are you still working on this PR?

**Comment by HWaruguru** [comment]
Hi @Damilare1 no I am done with it

**Comment by Damilare1** [comment]
@HWaruguru the Home tests are failing, could you resolve those and re-request review when you're done. Requesting review would help reviewers know when you're done or still working on a PR.

**Comment by HWaruguru** [comment]
Hi @Damilare I have rectified the test that was failing I have also added a test for the Skip to Next button. Please have a look thank you

**Review by Damilare1 — APPROVED** [review]
Thanks for all the work on this @HWaruguru. LGTM

Over to you @nbarnabee @blancadesal

**Review by nbarnabee — APPROVED** [review]
Looks good to me, too

**Pull request merged**